modules/swagger-codegen/src/main/resources/swift3/Configuration.mustache

```diff
@@ -0,0 +1,11 @@
+// Configuration.swift
+//
+// Generated by swagger-codegen
+// https://github.com/swagger-api/swagger-codegen
+//
+
+import Foundation
+
+open class Configuration {
+    open static var dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
```

Comment:
I understand you need it. I just think it's less headache for you to handle it on the server-side rather than try to patch a very complex client generator :) And I suspect you will eventually find yourself in a situation where you actually want ISO-8601 dates in this same client, but can no longer support that with this approach. 

For the time being though, I am fine with this if you just drop a comment on the `Configuration` property noting the value is only read once.